docs/source/predict/MaxCompute 离线预测.md

```diff
@@ -29,18 +29,18 @@ pai -name easy_rec_ext
 - selected_cols: 预测模型需要的columns，selected_cols和excluded_cols不能同时使用
 - reserved_cols: 需要copy到output_table的columns, 和excluded_cols/selected_cols不冲突，如果指定ALL_COLUMNS，则所有的column都被copy到output_table
 - batch_size: minibatch的大小
-- -Darn: rolearn  注意这个的arn要替换成客户自己的。可以从dataworks的设置中查看arn。
-- -DossHost: ossHost地址
-- -Dbuckets: config所在的bucket和保存模型的bucket; 如果有多个bucket，逗号分割
+- arn: rolearn  注意这个的arn要替换成客户自己的。可以从dataworks的设置中查看arn。
+- ossHost: ossHost地址
+- buckets: config所在的bucket和保存模型的bucket; 如果有多个bucket，逗号分割
 - 如果是pai内部版,则不需要指定arn和ossHost, arn和ossHost放在-Dbuckets里面
-  - -Dbuckets=oss://easyrec/?role_arn=acs:ram::xxx:role/ev-ext-test-oss&host=oss-cn-beijing-internal.aliyuncs.com
+  - -Dbuckets=oss://easyrec/?role_arn=acs:ram::xxx:role/aliyunodpspaidefaultrole&host=oss-cn-beijing-internal.aliyuncs.com
 - output_cols: output_name和类型, 如:
   - -Doutput_cols="probs double"
   - 如果有多列，用逗号分割, -Doutput_cols='probs double,embedding string'
```

Comment:
if it is multiclass, the output format is fixed, so user does not need to set the -Doutput_cols?